src/esw/brushed_motors/Controller.cpp

```diff
@@ -197,12 +198,34 @@ void Controller::turnOn() const {
 }
 
 // REQUIRES: nothing
-// MODIFIES: isLive
+// MODIFIES: liveMap
 // EFFECTS: I2C bus, if not already live,
 // configures the physical controller.
 // Then makes live.
+
+std::unordered_map<uint8_t, LiveState> Controller::liveMap;
+
 void Controller::makeLive() {
-    if (isLive) {
+    // if (isLive) {
+    //     return;
+    // }
+
+    uint8_t key = (deviceAddress << 3) | motorID;
```

Comment:
We should probably just make a function that does this. Seems like a bit of code that may need to get reused in the future